plugins/obs-qsv11/QSV_Encoder_Internal.cpp

```diff
@@ -328,6 +347,24 @@ mfxStatus QSV_Encoder_Internal::InitParams(qsv_param_t *pParams,
 		extendedBuffers.push_back((mfxExtBuffer *)&m_ExtAv1TileParam);
 	}
 
+#if MFX_VERSION_MAJOR >= 2 && MFX_VERSION_MINOR >= 11
+	PRAGMA_WARN_PUSH
+	PRAGMA_WARN_DEPRECATION
```

Comment:
Is there a reason this is also checked with preprocessor checks?

Reply:
The preprocessor check is for the m_ExtAV1ScreenContentTools variable. This check will be consistent with the check in the QSV_Encoder_Internal.h where the variable is defined.

Reply:
As well as the MFX_EXT_BUFF_AV1_SCREEN_CONTENT_TOOLS flag

Reply:
Got it.